### PR TITLE
update ppdet docs & remove static code, test=document_fix

### PR DESCRIPTION
**Pull request by jerrywgz**

### CODE REVIEW

**Review by qingqing01 — DISMISSED** [review]

File: README_cn.md [hunk 1]
```diff
@@ -353,19 +357,20 @@
...
-| **T4 TensorRT FP16 预测速度** | **28.0ms**                                                                             | **33.1ms**                                                                             | **单人2ms**                                                                                 | **单人2.9ms**                                                                               | **单人2.7ms**                                                       | **单人1.5ms**                                                            |
+| 任务            | 端到端速度（ms）|  模型方案  |  模型体积 |
+| :---------:     | :-------:  |  :------: |:------: |
+|  行人跟踪（高精度）  | 31.8ms  |  [多目标跟踪](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip) | 182M |  
+|  行人跟踪（轻量级）  | 21.0ms  |  [多目标跟踪](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_s_36e_pipeline.zip) | 27M |
+|  属性识别（高精度）  |   单人8.5ms | [目标检测](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip)<br> [属性识别](https://bj.bcebos.com/v1/paddledet/models/pipeline/strongbaseline_r50_30e_pa100k.zip) | 目标检测：182M<br>属性识别：86M |
```
Comment (zhiboniu) [inline README_cn.md]: 属性模型的链接换成新的吧，这个strongbaseline应该是v1的

**Review by nemonameless — DISMISSED** [review]

File: deploy/pipeline/README.md [hunk 2]
```diff
@@ -25,20 +26,45 @@ PP-Human支持图片/单镜头视频/多镜头视频多种输入方式，功能
...
-| 抽烟行为识别   |  视频输入 行为识别  | mAP: 39.7 |  单人2.0ms      | - |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/ppyoloe_crn_s_80e_smoking_visdrone.zip) |
-| ReID         | 视频输入 跨镜跟踪   | mAP: 98.8 | 单人1.5ms        | - |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/reid_model.zip) |
-
-下载预测部署模型并解压存放至`./output_inference`新建目录中
+| 目标检测(高精度) | 图片输入 | mAP: 57.8  | 31.8ms          | 182M |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip) |
+| 目标检测(轻量级) | 图片输入 | mAP: 53.2  | 21.0ms          | 27M |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_s_36e_pipeline.zip) |
+| 目标跟踪(高精度) | 视频输入 | MOTA: 82.2  | 31.8ms           | 182M |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip) |
+| 目标跟踪(轻量级) | 视频输入 | MOTA: 73.9  | 21.0ms           |27M |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_s_36e_pipeline.zip) |
+| 属性识别（高精度）    | 图片/视频输入 属性识别  | mA: 95.4 |  单人4.2ms     | 86M |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/strongbaseline_r50_30e_pa100k.zip) |
```
Comment (zhiboniu) [inline deploy/pipeline/README.md]: 同上，这里链接也更新一下吧，高精度和轻量级都是

File: deploy/pipeline/README.md [hunk 3]
```diff
@@ -25,20 +26,45 @@ PP-Human支持图片/单镜头视频/多镜头视频多种输入方式，功能
...
+</details>
+
+<details>
+<summary><b>端到端模型效果（点击展开）</b></summary>
+
+| 任务            | 端到端速度（ms）|  模型方案  |  模型体积 |
+| :---------:     | :-------:  |  :------: |:------: |
+|  行人跟踪（高精度）  | 31.8ms  |  [多目标跟踪](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip) | 182M |  
+|  行人跟踪（轻量级）  | 21.0ms  |  [多目标跟踪](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_s_36e_pipeline.zip) | 27M |
+|  属性识别（高精度）  |   单人8.5ms | [目标检测](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip)<br> [属性识别](https://bj.bcebos.com/v1/paddledet/models/pipeline/strongbaseline_r50_30e_pa100k.zip) | 目标检测：182M<br>属性识别：86M |
```
Comment (zhiboniu) [inline deploy/pipeline/README.md:58]: 同上

File: deploy/pipeline/docs/tutorials/QUICK_STARTED.md [hunk 4]
```diff
@@ -25,20 +39,21 @@ pip install -r requirements.txt
...
-| 关键点检测    | 视频输入 行为识别 | AP: 87.1 | 单人2.9ms        |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/dark_hrnet_w32_256x192.pdparams) |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/dark_hrnet_w32_256x192.zip)
-| 行为识别   |  视频输入 行为识别  | 准确率: 96.43 |  单人2.7ms      | - |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/STGCN.zip) |
-| ReID         | 视频输入 跨镜跟踪   | mAP: 98.8 | 单人1.5ms        | - |[下载链接](https://bj.bcebos.com/v1/paddledet/models/pipeline/reid_model.zip) |
+| 任务            | 端到端速度（ms）|  模型方案  |  模型体积 |
+| :---------:     | :-------:  |  :------: |:------: |
+|  行人跟踪（高精度）  | 31.8ms  |  [多目标跟踪](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip) | 182M |
+|  行人跟踪（轻量级）  | 21.0ms  |  [多目标跟踪](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_s_36e_pipeline.zip) | 27M |
+|  属性识别（高精度）  |   单人8.5ms | [目标检测](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip)<br> [属性识别](https://bj.bcebos.com/v1/paddledet/models/pipeline/strongbaseline_r50_30e_pa100k.zip) | 目标检测：182M<br>属性识别：86M |
```
Comment (zhiboniu) [inline deploy/pipeline/docs/tutorials/QUICK_STARTED.md]: 同上

File: deploy/pipeline/docs/tutorials/action.md [hunk 5]
```diff
@@ -152,34 +186,47 @@ python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_pph
...
+
+<div align="center">
+  <img src="../images/fight_demo.gif" width='1000'/>
+  <center>数据来源及版权归属：Surveillance Camera Fight Dataset。</center>
+</div>
+
+该方案关注的场景为监控摄像头下的打架行为识别。打架行为涉及多人，基于骨骼点技术的方案更适用于单人的行为识别。此外，打架行为对时序信息依赖较强，基于检测和分类的方案也不太适用。由于监控场景背景复杂，人的密集程度、光线、拍摄角度等都会对识别造成影响，本方案采用基于视频分类的方式判断视频中是否存在打架行为。针对摄像头距离人较远的情况，通过增大输入图像分辨率优化。由于训练数据有限，采用数据增强的方式提升模型的泛化性能。
+
+### 模型库
+在这里，我们提供了检测/跟踪、关键点识别、识别打架、打电话行为、抽烟行为、以及摔倒动作的预训练模型，用户可以直接下载使用。
```
Comment (zhiboniu) [inline deploy/pipeline/docs/tutorials/action.md]: 这句行为多了

File: deploy/pipeline/docs/tutorials/action.md [hunk 6]
```diff
@@ -121,12 +140,27 @@ python deploy/pipeline/pipeline.py --config deploy/pipeline/config/infer_cfg_pph
...
+
+应用行为：吸烟识别
+
 <div align="center">
   <img src="../images/smoking.gif" width='1000'/>
   <center>数据来源及版权归属：天覆科技，感谢提供并开源实际场景数据，仅限学术研究使用</center>
 </div>
 
+### 模型库
+在这里，我们提供了检测/跟踪、关键点识别、识别打架、打电话行为、抽烟行为、以及摔倒动作的预训练模型，用户可以直接下载使用。
```
Comment (zhiboniu) [inline deploy/pipeline/docs/tutorials/action.md]: 这里是不是只说跟踪、基于检测的行为就行了

File: docs/advanced_tutorials/openvino_inference/README_cn.md [hunk 7]
```diff
@@ -93,7 +93,7 @@ def predict(exec_net, input):
     return result
 ```
 
-您可能会惊讶地看到, 最激动人心的步骤居然如此简单。 不过下一个阶段会加复杂。 
+您可能会惊讶地看到, 最激动人心的步骤居然如此简单。 不过下一个阶段会加复杂。
```
Comment (zhiboniu) [inline docs/advanced_tutorials/openvino_inference/README_cn.md]: 更加

File: deploy/pipeline/README.md (hunk 2 above)
Comment (jerrywgz) [reply deploy/pipeline/README.md]: done，统一成zip格式了

File: deploy/pipeline/README.md (hunk 3 above)
Comment (jerrywgz) [reply deploy/pipeline/README.md:58]: done，thx

File: deploy/pipeline/docs/tutorials/QUICK_STARTED.md (hunk 4 above)
Comment (jerrywgz) [reply deploy/pipeline/docs/tutorials/QUICK_STARTED.md]: done，thx

File: docs/advanced_tutorials/openvino_inference/README_cn.md (hunk 7 above)
Comment (jerrywgz) [reply docs/advanced_tutorials/openvino_inference/README_cn.md]: done，thx

File: deploy/pipeline/docs/tutorials/action.md (hunk 5 above)
Comment (jerrywgz) [reply deploy/pipeline/docs/tutorials/action.md]: done thx

File: deploy/pipeline/docs/tutorials/action.md (hunk 6 above)
Comment (jerrywgz) [reply deploy/pipeline/docs/tutorials/action.md]: done thx

**Review by zhiboniu — APPROVED** [review]

**Review by nemonameless — APPROVED** [review]

**Review by zoooo0820 — APPROVED** [review]

**Comment by sfraczek** [comment]
Hi @jerrywgz. We are testing some models on our CI using static/deploy/python/infer.py. Since static is removed in release 2.5, Which app should we use now for testing both **qat** and **fp32** versions of:
faster_rcnn_r50_fpn_1x_coco
retinanet_r50_fpn_1x
yolov3_mobilenet
yolov3_darknet

replacing infer.py from static with this one deploy/python/infer.py results in errors. Could you help?
For example I run this:
```bash
python3.10 deploy/python/infer.py --model_dir=../retinanet_r50_fpn_1x/ --enable_mkldnn=True --run_benchmark=True --image_file=demo/000000014439.jpg
```
Which results in this:
```
------------------------------------------
Traceback (most recent call last):
  File "/mnt/drive/PaddlePaddle/PaddleDetection/deploy/python/infer.py", line 1034, in <module>
    main()
  File "/mnt/drive/PaddlePaddle/PaddleDetection/deploy/python/infer.py", line 968, in main
    detector = eval(detector_func)(
  File "/mnt/drive/PaddlePaddle/PaddleDetection/deploy/python/infer.py", line 103, in __init__
    self.pred_config = self.set_config(model_dir)
  File "/mnt/drive/PaddlePaddle/PaddleDetection/deploy/python/infer.py", line 126, in set_config
    return PredictConfig(model_dir)
  File "/mnt/drive/PaddlePaddle/PaddleDetection/deploy/python/infer.py", line 741, in __init__
    self.use_dynamic_shape = yml_conf['use_dynamic_shape']
KeyError: 'use_dynamic_shape'
```

**Pull request merged**